project/webapp/templates/partials/utils/civi.html

```diff
@@ -5,7 +5,7 @@
         <div class="col s10 no-pads civi-title-outer">
             <div class="civi-type gray-text">{{this.model.get('type')}}</div>
             <div class="civi-title ">
-                <span class="text-wrapper">{{this.model.get('title')}}</span>
+                <span class="text-wrapper" id = "first-civi">{{this.model.get('title')}}</span>
```

Comment:
Since we are working with templates, this will make it so that every civi has the `first-civi` id. Should be removed.